app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title><%= Rails.application.class.to_s.split("::").first %></title>
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <link rel="stylesheet" href="https://betterdoc-materialize.herokuapp.com/materialize/dist/css/betterdoc-materialize.min.css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link rel="stylesheet" href="https://unpkg.com/material-components-web@latest/dist/material-components-web.min.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,600,700,800" />
+    <link rel="stylesheet" href="https://betterdoc-materialize.herokuapp.com/mdc/dist/css/betterdoc-mdc.css">
+  </head>
+
+  <body>
+    <%= yield %>
+
+    <script src="https://betterdoc-materialize.herokuapp.com/materialize/dist/js/betterdoc-materialize.min.js"></script>
+    <script src="https://unpkg.com/material-components-web@latest/dist/material-components-web.min.js"></script>
+
+     <script>
+        if (window.M) {
```

Comment:
This should only be done `unless stacker_request?`

Reply:
Right, because Stacker will perform that automatically but the local development doesn't right?

Reply:
Exactly.

Reply:
Looks like my comment was not saved.

I don't think we need stacker request check here. We should only render inside layout if it is not stacker request and we do that check in controller already. For stacker request we return only plain html (without layout) anyway.

Reply:
Here is the relevant controller code https://github.com/betterdoc-org/general-gem-betterdoc-containerservice/blob/master/lib/betterdoc/containerservice/controllers/concerns/http_helpers_concern.rb#L18

Reply:
Better safe than not, right? :) It doesn't hurt to add it, I guess.

Reply:
If you return `<html><head>...</head><body>...</body></html>` you already introduced a bug so maybe it would be better then to add <%= raise "You are returning too much html to stacker" if stacker_request? %> to the top of layout file :)